site/data/docs/custom-wallet-list.mdx

```diff
@@ -145,6 +145,16 @@ coinbaseWallet(options: {
 });
 ```
 
+#### Enkrypt
+
+```tsx
+import { enkryptWallet } from '@rainbow-me/rainbowkit/wallets';
+enkryptWallet(options: {
+  chains: Chain[];
+  shimDisconnect?: boolean;
```

Comment:
For simplicity, we have deprecated `shimDisconnect` from the Custom Wallet List docs